draft-ietf-quic-multipath.md

```diff
@@ -1242,36 +1253,38 @@ Terrestrial | 100ms  | 350ms
 Satellite   | 350ms  | 600ms
 {: #fig-example-ack-delay title="Example of ACK delays using multiple paths"}
 
-The PATH_ACK frames describe packets that were sent on the specified path,
-but they may be received through any available path. There is an
-understandable concern that if successive acknowledgments are received
-on different paths, the measured RTT samples will fluctuate widely,
-and that might result in poor performance. While this may be a concern,
-the actual behavior is complex.
 
 The computed values reflect both the state of the network path and the
-scheduling decisions by the sender of the PATH_ACK frames. In the example
-above, we may assume that the PATH_ACK will be sent over the terrestrial
-link, because that provides the best response time. In that case, the
-computed RTT value for the satellite path will be about 350ms. This
-lower than the 600ms that would be measured if the PATH_ACK came over
+scheduling decisions of the acknowledgement sender. If we
+assume that the PATH_ACK will be sent over the terrestrial
+link, because this decision provides the best response time, the
+computed RTT value for the satellite path will be about 350ms. This is
+lower than the maximum of 600ms that would be measured if the PATH_ACK came over
 the satellite channel, but it is still the right value for computing
 for example the PTO timeout: if a PATH_ACK is not received after more
 than 350ms, either the data packet or its PATH_ACK were probably lost.
 
-The simplest implementation is to compute smoothed_rtt and rttvar per
+The simplest implementation is to use the acknowledgment delay
+to compute smoothed_rtt and rttvar per
 {{Section 5.3 of QUIC-RECOVERY}} regardless of the path through which PATH_ACK frames are
-received. This algorithm will provide good results,
-except if the set of paths changes and the PATH_ACK sender
-revisits its sending preferences. This is not very
-different from what happens on a single path if the routing changes.
+received. This approach will provide good results
+as long as acknowledgements are sent consistently over the same paths.
+If the acknowledgement sender
+revisits its sending preferences, the set of paths can change. However, this is not very
```

Comment:
Maybe a suggestion, but if this does not work (as it is longer) we can stick with the current text
```suggestion
However, endpoints may at some point revisit sending preferences, leading to
other paths being used to send acknowledgements. However, this is not very
```

Reply:
```suggestion
If at any time the sender revisits its sending preferences,
this can also change the paths that are used to send acknowledgements.
However, this is not very
```

Reply:
@qdeconinck Is this okay?

Reply:
@mirjak Yes, let's take this last version.